.browserslistrc

```diff
@@ -0,0 +1,8 @@
+last 2 Chrome versions
```

Comment:
isn't not dead enough ?

Reply:
Good point, `> 0.25%, not dead` is enough. Updated in https://github.com/Open-Attestation/decentralized-renderer-react-components/pull/62/commits/c78f63282e44691283f6363d8adedef5df810d2a.